src/main/java/com/example/springbootbookshop/dto/BookDto.java

```diff
@@ -0,0 +1,7 @@
+package com.example.springbootbookshop.dto;
+
+import java.math.BigDecimal;
+
+public record BookDto(Long id, String title, String author,
+                      String isbn, BigDecimal price, String description, String coverImage) {
```

Comment:
minor

fields records are usually placed separately:
```suggestion
public record BookDto(Long id, 
                      String title, 
                      String author,
                      String isbn, 
                      BigDecimal price, 
                      String description, 
                      String coverImage) {
```